tensorflow/python/framework/errors_impl.py

```diff
@@ -283,7 +283,7 @@ def __init__(self, node_def, op, message):
 
 
 @tf_export("errors.NotFoundError")
-class NotFoundError(OpError):
+class NotFoundError(OpError, FileNotFoundError):
```

Comment:
Hmm, we are returning `NotFoundError` in other cases too. See for example https://cs.opensource.google/tensorflow/tensorflow/+/master:tensorflow/core/framework/op_kernel.cc;l=1417;drc=a565c473c14da0c1bb35bd1771bdc195a72f7982